Thrive.csproj

```diff
@@ -232,7 +233,7 @@
     <Compile Include="src\gui_common\CompoundListBox.cs" />
     <Compile Include="src\gui_common\CustomCheckBox.cs" />
     <Compile Include="src\gui_common\CreditsScroll.cs" />
-    <Compile Include="src\gui_common\CustomDialog.cs" />
+    <Compile Include="src\gui_common\dialogs\CustomDialog.cs" />
```

Comment:
I keep losing track of where this class is, so this should make it easier to find.